src/main/utils.ts

```diff
@@ -41,7 +43,11 @@ export function openLogsDirectory() {
   const logDirectory = path.dirname(log.transports.file?.getFile()?.path);
 
   if (!logDirectory) {
-    log.error('Could not find log directory!');
```

Comment:
Looks like it helped with consistently immediately, too :)

Reply:
Absolutely! Great suggestion